full-service/src/service/sync.rs

```diff
@@ -230,6 +230,17 @@ pub fn sync_account_next_chunk(
             (*account_key.view_private_key(), Some(account_key))
         };
 
+        tx_outs.iter().try_for_each(|(block_index, tx_out)| {
```

Comment:
Moved the logic for marking transaction logs as succeeded so that the `txo_outs` didn't need to be cloned, as they are consumed by the `into_par_iter()` below.

I don't think there is any intentional rhyme or reason on the order of operations, in particular, with regard to potential errors that may occur.